public/app/features/explore/Wrapper.tsx

```diff
@@ -14,6 +15,20 @@ import { ExploreActions } from './ExploreActions';
 import { ExplorePaneContainer } from './ExplorePaneContainer';
 import { lastSavedUrl, resetExploreAction, richHistoryUpdatedAction } from './state/main';
 
+const getStyles = () => {
+  return {
+    pageScrollbarWrapper: css`
+      width: 100%;
+      flex-grow: 1;
+      min-height: 0;
+    `,
+    exploreWrapper: css`
+      display: flex;
+      height: 100%;
+    `,
+  };
+};
+
```

Comment:
nit: we don't need this to be a function, it can just be an object here:
```suggestion
const styles = {
  pageScrollbarWrapper: css`
    width: 100%;
    flex-grow: 1;
    min-height: 0;
  `,
  exploreWrapper: css`
    display: flex;
    height: 100%;
  `,
};
```

and you can remove the function call in https://github.com/grafana/grafana/pull/52078/files#diff-06aad0593d46c656f9c92b1a6e517f30e9758c1f1eff827194d81389ee5c74f6R85